KNN/Network_Base.cpp

```diff
@@ -27,10 +28,10 @@ Network_Base::Network_Base(size_t dz, size_t iz, size_t oz, unique_ptr<float[]>
 	}
 
 	//Lernen
-
-	while (output_layer[0].adjust_weights(data[data_size - 1], input_size) > 5.28419e-06) {	};
+	while (output_layer[0].adjust_weights(data[data_size - 1], input_size) > 5.28419e-06)
+	{
+	};
```

Comment:
Same here with the brackets. There's nothing wrong with this code but I'd love to keep it all in one style